core/service/section/vms-section-service.js

```diff
@@ -0,0 +1,437 @@
+var AbstractSectionService = require("core/service/section/abstract-section-service").AbstractSectionService,
+    VmGuestType = require("core/model/enumerations/vm-guest-type").VmGuestType,
+    VmConfigBootloader = require("core/model/enumerations/vm-config-bootloader").VmConfigBootloader,
+    VmDeviceDiskMode = require("core/model/enumerations/vm-device-disk-mode").VmDeviceDiskMode,
+    VmDeviceGraphicsResolution = require("core/model/enumerations/vm-device-graphics-resolution").VmDeviceGraphicsResolution,
+    VmDeviceNicDevice = require("core/model/enumerations/vm-device-nic-device").VmDeviceNicDevice,
+    VmDeviceNicMode = require("core/model/enumerations/vm-device-nic-mode").VmDeviceNicMode,
+    VmDeviceUsbDevice = require("core/model/enumerations/vm-device-usb-device").VmDeviceUsbDevice,
+    VmDeviceVolumeType = require("core/model/enumerations/vm-device-volume-type").VmDeviceVolumeType,
+    VmRepository = require("core/repository/vm-repository").VmRepository,
+    StorageRepository = require("core/repository/storage-repository").StorageRepository,
+    NetworkRepository = require("core/repository/network-repository").NetworkRepository,
+    BytesService = require("core/service/bytes-service").BytesService,
+    ConsoleService = require("core/service/console-service").ConsoleService,
+    Dict = require("collections/dict").Dict;
+
+exports.VmsSectionService = AbstractSectionService.specialize({
+    DEFAULT_STRING: {
```

Comment:
I have seen multiple occurence of this string within GUI, are going to have a global constants file?